### DISCUSSION TITLE
Sonnet 3.5 Token Limit at 8192?

### DISCUSSION THREAD

**Discussion by srcnkmn** [danny-avila/LibreChat#5083] [General]
Hi,

I am very happy with librechat and am playing around all the time to see its capabilities.

Since I installed the Anthropic API Key I was wondering whether sonnet 3.5 has the max token limit set to 8192 or 4096.

There seems to be "hack" to activate the higher token limit, see https://www.reddit.com/r/ClaudeAI/s/OHNJnddvjg

Is this set? How can I check or edit that? :
Thanks a lot in advance 

Best regards

**Comment by danny-avila** [comment]
Glad you are enjoying LibreChat! It’s already set when using the regular endpoint option:

https://github.com/danny-avila/LibreChat/blob/9b118d42de3f99e9c8c85d0beb46d4ae5fd74a4c/api/app/clients/AnthropicClient.js#L182-L184

but will need to add it to Anthropic [agents](https://librechat.ai/docs/features/agents)

**Reply by srcnkmn** [reply]
Thank you :)
Now I have generated an agent with sonnet-3-5-latest

Is there anything else to do? Sorry for being a noob right now with Agents
